content/en/docs/v3.4/op-guide/configuration.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Configuration flags
+title: Configuration options
```

Comment:
Minor comment, I am not sure whether we need to change `flags` to `options`, because `flag` or `flags` is used in the entire doc.

Reply:
The reason I thought we should change to `options` because there are three ways we mention for configuration: config flags, config file, and env variables. So, I found options more broader vs flags. I am okay to change it, if you still think so? Thanks!

Reply:
Sounds good to me.

Reply:
Thanks for the quick look :) @ahrtr